binding_generator.py

```diff
@@ -342,6 +343,101 @@ def generate_builtin_bindings(api, output_dir, build_config):
 
         builtin_binds_file.write("\n".join(builtin_binds))
 
+    # Craete a header to implement all builin class vararg methods and be included in `variant.hpp``.
```

Comment:
Some minor typos here, like `Craete`, `builin` and a redundant backtick at the end.

Reply:
Thanks for your warning.